libs/utils/helpers.js

```diff
@@ -19,3 +19,9 @@ export function updateLinkWithLangRoot(link) {
     return link;
   }
 }
+
+export async function fetchWithCacheRules(path) {
+  const params = new URLSearchParams(window.location.search);
+  const cache = params.get('cache') === 'off' ? 'reload' : 'default';
+  return fetch(path, { cache });
+}
```

Comment:
Given the [global nav already has a use-case](https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L301) to do _something_ after fetching which will eventually be ported to a wider range of blocks that could to use this feature, it might make sense to make the fetch a little more generic and allow to plug a few more features as a great spot in the future. Wdyt?
```suggestion
// Makes this method extendable
export async function customFetch({resource, withCacheRules}) {
  const cache = 'default';
  if(withCacheRules) {
    const params = new URLSearchParams(window.location.search);
    params.get('cache') === 'off' ? 'reload' : 
  }
  // if(fetchCentralContent) {
  // apply custom logic
  // }
  return fetch(resource, { cache });
}
```